include/uapi/linux/videodev2.h

```diff
@@ -738,6 +738,8 @@ struct v4l2_pix_format {
 #define V4L2_PIX_FMT_INZI     v4l2_fourcc('I', 'N', 'Z', 'I') /* Intel Planar Greyscale 10-bit and Depth 16-bit */
 #define V4L2_PIX_FMT_SUNXI_TILED_NV12 v4l2_fourcc('S', 'T', '1', '2') /* Sunxi Tiled NV12 Format */
 #define V4L2_PIX_FMT_CNF4     v4l2_fourcc('C', 'N', 'F', '4') /* Intel 4-bit packed depth confidence information */
+#define V4L2_PIX_FMT_SAND8    v4l2_fourcc('S', 'A', 'N', 'D') /* RPI 8 bit sand */
```

Comment:
What was wrong with the V4L2_PIX_FMT_NV12_COL128 format I'd defined in the earlier patch?

Reply:
> What was wrong with the V4L2_PIX_FMT_NV12_COL128 format I'd defined in the earlier patch?

Probably nothing - either I wrote that line before I found your patch or I failed to notice it.